hack/quick-install.sh

```diff
@@ -26,12 +28,21 @@ data:
 EOF
 
 bash cmd/operator/generate.sh
+helm template kubearchive charts/kubearchive -n kubearchive \
+    --include-crds \
+    --set "global.production=true" \
+    --set "database.enabled=false" > /tmp/kubearchive-not-resolved.yaml
+ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > kubearchive.yaml
+kubectl apply -n kubearchive -f kubearchive.yaml
```

Comment:
I noticed that the files you are creating in this script are going into `/tmp` except for `kubearchive.yaml`. Is keeping the `kubearchive.yaml` file in the current directory after running the script intentional? I ask because the `release.sh` script removes the `kubearchive.yaml` file when the script is complete.

Reply:
Its not really intentional, as there is no point of keeping it in the current directory. I will send it to `/tmp/` too. If someone wants to inspect it, they can open it directly from `/tmp/`.